auth-server/src/services/auth.ts

```diff
@@ -0,0 +1,126 @@
+import * as jwt from 'jsonwebtoken'
+import { NotFoundError } from 'objection'
+import { DataError } from 'objection-db-errors'
+import { uuid } from 'uuidv4'
+
+import User from '../model/User'
+import RefreshToken from '../model/RefreshToken'
+
+export default class AuthService {
+	constructor(){}
+
+	public async Login(username, password): Promise<any> {
```

Comment:
Could we create a `responseType` so that we don't `any` here?

Reply:
done